src/components/fileUpload/resume.tsx

```diff
@@ -73,11 +74,8 @@ const ResumeUpload: React.FC<ResumeUploadProps> = ({ user_id, url }) => {
       });
 
       if (putResponse.status === 200) {
-        setPreviewUrl(
-          selectedResume.type === 'image/png'
-            ? URL.createObjectURL(selectedResume)
-            : Location,
-        ); // Update previewUrl based on file type
+        setPreviewUrl(selectedResume.type === 'image/png' ? Location : null);
```

Comment:
Why is there check for selectedResume against 'image/png' when at upload prompt it says 'Supported formats: JPG, PNG,JPEG'?
one of these are incorrect, please find out which and fix it